source/triggers.txt

```diff
@@ -1,22 +1,105 @@
-========
-Triggers
-========
+.. _triggers:
+.. _triggers-overview:
 
-.. default-domain:: mongodb
+==============
+Atlas Triggers
+==============
 
 .. toctree::
    :titlesonly:
    :caption: Triggers
    :hidden:
 
-   About Triggers </triggers/overview>
    Database Triggers </triggers/database-triggers>
    Authentication Triggers </triggers/authentication-triggers>
    Scheduled Triggers </triggers/scheduled-triggers>
-   Usage Examples </triggers/examples>
+   Resume a Suspended Trigger </triggers/resume-a-suspended-trigger>
+   Disable a Trigger </triggers/disable>
```

Comment:
Fine to have this as a separate page for now - eventually it'd be great to have this on a "manage triggers" page or somesuch that collects all the various CRUD/admin tasks.